end-to-end-test/local/docker/catalina_server.xml.patch

```diff
@@ -0,0 +1,11 @@
+--- server.xml.default	2016-07-18 15:10:55.000000000 +0200
```

Comment:
this we don't need anymore

Reply:
Docker solution was set to the new docker solution. File was removed.